src/navbarList.js

```diff
@@ -0,0 +1,32 @@
+export default [
+  {
```

Comment:
it can be in JSON format and inside component's folder

Reply:
changed location